RELEASE_NOTES.md

```diff
@@ -103,6 +170,1119 @@ For a list of all features with an announced removal target, see [Removals](intr
     * A workaround is presented in [IUF does not run the next stage for an activity](troubleshooting/known_issues/iuf_unable_to_run_next_stage.md)
 * iSCSI based boot content projection may fail
     * If the image to be projected does not have an `etag`
-    * A workaround is presented in [iSCSI SBPS boot failure](https://github.com/Cray-HPE/docs-csm/blob/release/1.6/troubleshooting/known_issues/SBPS_boot_fail.md)
+        * A workaround is presented in [iSCSI SBPS boot failure](https://github.com/Cray-HPE/docs-csm/blob/release/1.6/troubleshooting/known_issues/SBPS_boot_fail.md)
```

Comment:
I don't think this should be indented over further. Also, the link should be relative.
```suggestion
    * A workaround is presented in [iSCSI SBPS boot failure](troubleshooting/known_issues/SBPS_boot_fail.md)
```